README.md

```diff
@@ -120,16 +111,16 @@ GitHub.authorize_url!
 Capture the `code` in your callback route on your server and use it to obtain an access token.
 
 ```elixir
-token = GitHub.get_token!(code: code)
+client = GitHub.get_token!(code: code)
```

Comment:
@jc00ke yeah, I've been going back and forth on this one...probably will end up just returning the token separately...allowing the user to then create a new client by passing in the token...


```elixir
client = GitHub.new
token = GitHub.get_token!(client, code: "abc")
client = GitHub.new(client, token: token)
```